web/src/cypress/integration/profile.ts

```diff
@@ -99,6 +99,26 @@ function testProfile(): void {
       cy.get('body').should('contain', `${name} (${type})`)
     })
 
+    it('should return error with link to conflicting user', () => {
```

Comment:
One thing we can do that I didn't think of while we were pairing is to use the 'users' fixture instead of the 'profile' fixture to create the first contact method. This would allow us to verify that clicking the link actually works (if we were to test clicking the link as is nothing would happen but the dialog closing since it's the same profile).

it would look something along the lines of:

```
cy.fixture('users').then(users => {
  cy.addContactMethod({ userID: users[0].id }).then(
    ...
    ...
    cy.dialog().contains('Contact method already exists for that type and value: ' + users[0].name).click()
    cy.get('[data-cy="details-heading"]').should('contain', users[0].name)
  })
})
```